build.linux32x86/editpharoinstall.sh

```diff
@@ -11,8 +11,11 @@ if [ "$1" = -source ]; then
 else
 	SourceFile="PharoV50"
 fi
-SOURCE=../../sources/$SourceFile.sources
-test -f $SOURCE || SOURCE=../../../sources/$SourceFile.sources
```

Comment:
The root directory seems like the best place for the 'sources' folder. There are multiple build.xxx folders that use the files. That should be okay since the .sources files are gitignored.